lib/scopes/form526_submission_state.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+
+module Scopes
+  module Form526SubmissionState
+    extend ActiveSupport::Concern
+
+    # DOCUMENTATION:
+    # https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/engineering_research/526_scopes.md
+    included do
+      scope :pending_backup, lambda {
+        where(submitted_claim_id: nil, backup_submitted_claim_status: nil)
+          .where.not(backup_submitted_claim_id: nil)
+          .where.missing(:form526_submission_remediations)
+          .where(arel_table[:created_at].gt(Form526Submission::MAX_PENDING_TIME.ago))
+      }
+      scope :in_process, lambda {
+        where(submitted_claim_id: nil)
+          .where(backup_submitted_claim_id: nil)
```

Comment:
Has the scope for `in_process` changed? Previously it was scoped to `.where(backup_submitted_claim_status: nil)` instead of `.where(backup_submitted_claim_id: nil)`.

Reply:
yes, check out the doc for a breakdown but TL;DR - in_process means it hasn't  been accepted to either path yet. The scope that cares about  `backup_submitted_claim_status` is `pending_backup`. these two are now wrapped up together in `incomplete_type`